horizons/world/player.py

```diff
@@ -38,7 +38,16 @@
 
 
 class Player(ComponentHolder, WorldObject):
-	"""Class representing a player"""
+	"""Class representing a player
+
+	@type session: horizons.session.Session
+	@param session: Session instance
+	@param worldid: player's worldid
+	@param name: user-chosen name
+	@param color: color of player (as Color)
+	@param clientid: id of client
+	@param inventory: {res: value} that are put in the players inventory
+	"""
```

Comment:
Your shell is probably complaining about quotes. Make sure to wrap those """ in quotes again, like this:

`git grep -A 1 class | grep _B 1 '"""'` (note the ')